locksmith/__tests__/operations/userOperations.test.ts

```diff
@@ -20,14 +21,14 @@ describe('User creation', () => {
 
       await UserOperations.createUser(userCreationDetails)
```

Comment:
Could you add a mock on `RecoveryPhrase.generate()` so that we know that the string below is actually coming from it?

Reply:
yes, i'll adjust this